dist_tables/ddl.rst

```diff
@@ -61,6 +61,45 @@ Each created shard is assigned a unique shard id and all its replicas have the s
 
 You are now ready to insert data into the distributed table and run queries on it. You can also learn more about the UDF used in this section in the :ref:`user_defined_functions` of our documentation.
 
+.. _colocation_groups:
+
+Co-Location Groups
```

Comment:
Do we want to extend on this section by first explaining what table colocation is?

* Table co-location: What is table colocation and an example of it happening
   * Co-location groups

When I read the first paragraph, it reads a bit dense and direct. What if we started with what table colocation is and why you would want it. (We talk on the motivation for table colocation in the second paragraph.)